src/Core/Analysis/IAnalysisIssue.cs

```diff
@@ -55,14 +55,29 @@ public interface IAnalysisIssueLocation : ITextRange
         string Message { get; }
     }
 
+    /// <summary>
+    /// Represents a Sonar text range
+    /// </summary>
     public interface ITextRange
     {
+        /// <summary>
+        /// 1-based line
+        /// </summary>
         int StartLine { get; }
 
+        /// <summary>
+        /// 1-based line
+        /// </summary>
         int EndLine { get; }
 
+        /// <summary>
+        /// 0-based column
```

Comment:
We should be consistent and have everything either 1 based or 0 based. I've documented the existing behavior.